[TW-1531] Send requests split (#5403)
postmanlabs/postman-docs: src/pages/docs/sending-requests/variables/managing-environments.md

1. Select a variable **Type**. If you select **default** the variable value is visible in plain text. If you select **secret** the variable value is masked.

>     > If a variable's initial value has sensitive data, make sure to choose the **secret** variable type if you don't want to expose the value when sharing or publishing the environment. Learn more about [variable types](/docs/sending-requests/variables/variables/#variable-types).

Reviewer: ```suggestion
    > If a variable's initial value has sensitive data, select the **secret** variable type to prevent exposing the value when sharing or publishing the environment. Learn more about [variable types](/docs/sending-requests/variables/variables/#variable-types).
```

<img alt="Environment quick look" src="https://assets.postman.com/postman-docs/v10/environment-quick-look-selector-v10-20.jpg"/>

> > If you want to make variables available no matter which environment is active, use [global variables](/docs/sending-requests/variables/variables/#defining-global-variables). The quick look lists any global variables you have declared or that are shared in your workspace.

Reviewer: ```suggestion
> If you want to make variables available regardless of which environment is active, use [global variables](/docs/sending-requests/variables/variables/#defining-global-variables). The quick look lists any global variables you have declared or that are shared in your workspace.
```

<img alt="Environment variable reference" src="https://assets.postman.com/postman-docs/v10/environment-hover-v10-20.jpg" width="580px"/>

You can access current environment variable values in your **Pre-request Script** and **Tests** code using the [`pm.environment.get` method](/docs/sending-requests/variables/variables/#using-variables-in-scripts):

```js
pm.environment.get("variable_name");
```

> If more than one variable with the same name is available to a request, Postman will use the value from the variable with narrowest [scope](/docs/sending-requests/variables/variables/#variable-scopes). This means that if you have an environment variable with the same name as a collection or global variable, Postman will use the environment variable (but local and data variable values will supersede environment values). The value of any overridden variables will display in strikethrough format.

Reviewer: ```suggestion
If more than one variable with the same name is available to a request, Postman will use the value from the variable with the narrowest [scope](/docs/sending-requests/variables/variables/#variable-scopes). Therefore, if you have an environment variable with the same name as a collection or global variable, Postman will use the environment variable, but local and data variable values will supersede environment values. The value of any overridden variables will display in strikethrough format.
```
